### ISSUE TITLE
Add total to summary information of Authorizations table for multi-currency stores.

### ISSUE THREAD

**Issue by mgascam** [Automattic/woocommerce-payments#5178]
### Description

In the first implementation, the total amount of transactions in the Uncaptured tab is hidden when the store supports multi-currency. We want to improve it to show the totals also in this case.
### Acceptance criteria

* Display total amount of Uncaptured transactions by currency

### Designs

TBD

### Dev notes

It is very likely that we will need to modify the server endpoint to return the totals by currency. I am not creating an issue yet, since the design is TBD and we do not know all the requirements.

### Additional context

P2: paJDYF-4N1-p2#comment-14033

**Comment by kalessil** [comment]
@mgascam: can you please re-evaluate if totals are listed in the right currency when we filter by no-USD one? Should be aligned with transactions listing in this regards.

**Comment by mgascam** [comment]
The issue is still relevant since totals are not listed for multi-currency

<img width="1702" alt="Screenshot 2024-02-09 at 18 55 24" src="https://github.com/Automattic/woocommerce-payments/assets/1553182/c7947df7-dde3-41a6-b67d-8e1cb4567611">

**Comment by kalessil** [comment]
Thanks for re-evaluating @mgascam ! Marking as low-priority as no requests to add this functionality, but keep in backlog as I'd expect the authorizations summary aligned with the transactions summary in terms of functionality.

**Comment by KarlisJ** [comment]
Looking at the related additional context aligning with the transactions summary would mean adding filters to the authorizations. Is that the way we want to go?

**Comment by kalessil** [comment]
@KarlisJ, correct. The ticket shall cover (can be broken down into smaller issues if necessary)
- 'Deposit currency' filter introduction
- changes in listing and summary to support the filtering
